Script.md

```diff
@@ -0,0 +1,11 @@
+## Introducing Lodgel – the property management application that streamlines the rental process.
+
+Users register for an account, and right away, they can start adding properties to their list.
+To ensure each guest is taken care of, checklists can be made that detail tasks that need to be taken care of
+before, during, and after each visit. 
+
+Assistants are an essential element of every property management business. Each property has an assistant 

```

Comment:
*Turnover assistants

Reply:
Maybe not every property management business, but holiday rentals 🤷‍♀️? Not sure here.